ftlib/__init__.py

```diff
@@ -3,19 +3,29 @@
 import logging
 import threading
 
+# in the ftlib package, user is able to initialize the package
+# with specific consensus method adn framework
+
 from .consensus.consensus_status import ConsensusStatus
 from .consensus.shared_storage import SharedStorage
 from .framework.framework_status import FrameworkStatus
-from .ftlib_status import FTAllReduceStatus, FTRebuildStatus
+from .ftlib_status import *
```

Comment:
There may be problems with CI or coding style checks. Can we use `FTAllReduceStatus, FTRebuildStatus`?

Reply:
fixed